src/lang/window.js

```diff
@@ -22,6 +22,19 @@
      */
     window.me = window.me || {};
 
+    /**
+    * Empty BaseClass constructor and prototype attached to 
+    * `me` global object to act as the base class for inheritance 
+    */
+    me.BaseClass = function () {};
+    me.BaseClass.prototype = Object.create(Object);
```

Comment:
Okay, the update is ready to go!  It seems like it performs slight better, too for some reason...